### ISSUE TITLE
Audit Algorithm too strict 

### ISSUE THREAD

**Issue by garandal245** [carefree0910/carefree-creator#19]
with the prompt of `2b, neir` it triggers the audit algorithm roughly 2/3 times is there any way to bypass this for the locally run server

**Comment by carefree0910** [comment]
Hi! 

**tl; dr**: You can actually access the generated images through the `Network` panel in the browser's console. 🤣

For now, I'm still storing the images on my poor cloud server, which requires me to do strict audit. 😣

But there's a fun fact: if you open the console of your browser and go to the `Network` panel, you can actually see that the image has already been generated and transferred to the webui, after which I will upload it to my poor cloud server, which triggers the audit algorithm.

So, the audit algorithm only stops you from uploading the output to my poor cloud server, but it does not stop you from generating! 😉
